requirements.json

```diff
@@ -23,7 +23,8 @@
       "javaDev": { },
       "fuseDev": { },
       "rhProds": { },
-      "devstudio": { }
+      "devstudio": { },
+      "all": { }
```

Comment:
Why are we adding all object in all the components. Any specific reason for this?

Reply:
`node.channel.hasOwnProperty(this.channel_tab)` code return true if it's find oblect. When we select `All` channel `node.channel.hasOwnProperty(this.channel_tab)`  return false because it's not able to find the object that is why I have added the all object in all components.

Reply:
This was working before when we do not use the `all` object. We need to find a way to use  without using `hasOwnProperty`.